aggregator/apps/api/v1/views.py

```diff
@@ -39,6 +46,14 @@ def get_response(self, request, *args, **kwargs):
 
         stitcher = Stitcher(wdiv, wcivf, request)
 
+        # Log to firehose
+        entry = settings.POSTCODE_LOGGER.entry_class(
```

Comment:
Feels slightly strange to me to see methods get called on a setting - an alternative might be to have the logger client initialised in an app config file perhaps, that can then be imported elsewhere? But also dont think it is too important and I understand what is happening

Reply:
I agree. I couldn't really find a good pattern to have a singleton / global var other than settings. Maybe app configs are the answer?

Reply:
Or possibly in an `__init__.py` file or a new `logger.py` file? I'm not too sure what is best either really! I think in the past I have had a `client.py` file or something similar but its always been more app specific rather than intended to be used globally so slightly different.